prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, audit_info):
         self.account = audit_info.audited_account
         self.audit_resources = audit_info.audit_resources
         self.partition = audit_info.audited_partition
+        self.account_arn = f"arn:{self.partition}:iam::{self.account}:root"
```

Comment:
Why you are not using `audited_account_arn` here?

Reply:
Fixed